pkgs/top-level/aliases.nix

```diff
@@ -164,6 +164,7 @@ mapAliases ({
   calibre-py3 = throw "calibre-py3 has been removed from nixpkgs, as calibre's default python version is now 3. Please use calibre as replacement"; # Added 2021-01-13
   callPackage_i686 = pkgsi686Linux.callPackage;
   cantarell_fonts = throw "'cantarell_fonts' has been renamed to/replaced by 'cantarell-fonts'"; # Converted to throw 2022-02-22
+  cantor = libsForQt5.cantor; # Added 2022-06-30
```

Comment:
The `aliases.nix` file is only for compatibility aliases. These are new packages, so there is no compatibility issue. These aliases should go in `all-packages.nix` so that they aren't deleted when they expire.